resources/assets/js/components/atoms/GridArea.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
Grid area is derived from the CSS syntax `grid-area`. But indeed not 100%. In this case, the dashboard component should be also named `Grid`. Renaming to `Tile`